docs/connect.html

```diff
@@ -129,9 +129,11 @@ <h4><a id="connect_transforms" href="#connect_transforms">Transformations</a></h
     <p>The file source connector reads each line as a String. We will wrap each line in a Map and then add a second field to identify the origin of the event. To do this, we use two transformations:</p>
     <ul>
         <li><b>HoistField</b> to place the input line inside a Map</li>
-        <li><b>InsertField</b> to add the static field. In this example we'll indicate that the record came from a file connector</li>
```

Comment:
This is also not correct. 
This refers to what we say above as: _then add a second field to identify the origin of the event_
So this correctly refers to what will be added in this new field when using `InsertField`

Reply:
Yes, you're right! After 2nd reading, it indeed refer to the content coming from a file in `InsertField`. I'll revert this change back. Thanks.